packages/wpcom.js/test-need-complete-access-origin/test.wpcom.site.settings.js

```diff
@@ -1,61 +1,72 @@
+/* eslint-disable jest/expect-expect */
```

Comment:
Everything in this file except the test description name change are auto-formatting